terraform/state-persistence.tf

```diff
@@ -0,0 +1,9 @@
+data "terraform_remote_state" "gt-site-cluster" {
```

Comment:
> This resource is setup to interact with Terraform state pushed and pulled primarily by another project.

What do you mean, exactly? I changed the critical names to avoid clashes with the existing site state.